### PR TITLE
use ternary operator instead of if else block

### PR DESCRIPTION
**Pull request by chukitow**
I think that is cleaner use ternary operator instead of **if else  block**  for this simple condition.

### CODE REVIEW

File: frontend/app/views/spree/products/_cart_form.html.erb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <% end %>
 
     <% if @product.price_for(current_pricing_options) and !@product.price.nil? %>
-      <div data-hook="product_price" class="columns five <% if !@product.has_variants? %> alpha <% else %> omega <% end %>">
+      <div data-hook="product_price" class="columns five <%= @product.has_variants? ? 'alpha' : 'omega' %>">
```
Comment (mamhoff) [inline frontend/app/views/spree/products/_cart_form.html.erb]: I think this is a fine change in principle, but the previous code had a `!` before the main condition, turning the two branches around. Maybe rather: 

```ruby
@product.has_variants? ? 'omega' : 'alpha'
```

Comment (chukitow) [inline frontend/app/views/spree/products/_cart_form.html.erb]: You are right, I thought this was going to be my first PR without comments hahaha, I'll change it ASAP

Comment (cbrunsdon) [inline frontend/app/views/spree/products/_cart_form.html.erb]: ha, nice catch Martin.

**Comment by chukitow** [comment]
I already did the change that you requested @mamhoff 😬

**Pull request merged**